src/accelerate/utils/imports.py

```diff
@@ -377,19 +377,28 @@ def is_npu_available(check_device=False):
 
 @lru_cache
 def is_xpu_available(check_device=False):
+    """
+    Checks if XPU acceleration is available either via `intel_extension_for_pytorch` or via stock PyTorch (>=2.4) and
+    potentially if a XPU is in the environment
+    """
+
     "check if user disables it explicitly"
     if not parse_flag_from_env("ACCELERATE_USE_XPU", default=True):
         return False
-    "Checks if `intel_extension_for_pytorch` is installed and potentially if a XPU is in the environment"
-    if is_ipex_available():
-        import torch
 
+    if importlib.util.find_spec("torch") is None:
+        return False
+
+    import torch
```

Comment:
This part can actually be removed, as accelerate always requires PyTorch :)

Reply:
Indeed:
https://github.com/huggingface/accelerate/blob/91a2599f933fa4ee7eda89e6b4f405d897a30367/src/accelerate/utils/imports.py#L21

I copied this from `is_npu_available` above. Which by the way also re-imports `torch`. Do you want me to also fix `is_npu_available()` in this PR?
https://github.com/huggingface/accelerate/blob/91a2599f933fa4ee7eda89e6b4f405d897a30367/src/accelerate/utils/imports.py#L362-L367

Reply:
> This part can actually be removed, as accelerate always requires PyTorch :)

Fixed. I will submit npu/mlu in a separate cleanup PR unless you will tell me otherwise.

Reply:
> I will submit npu/mlu in a separate cleanup PR unless you will tell me otherwise.

Submitted #2856.